mbtiles/src/mbtiles.rs

```diff
@@ -39,6 +39,60 @@ pub struct Metadata {
     pub json: Option<JSONValue>,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct LevelDetail {
+    pub zoom: String,
+    pub count: u32,
+    pub smallest: f32,
+    pub largest: f32,
+    pub average: f32,
+    pub bounding_box: [f32; 4],
+}
+
+#[derive(Clone, Debug, PartialEq, Serialize)]
+pub struct Statistics {
+    pub file_path: String,
+    pub file_size: f64,
+    pub schema: String,
+    pub page_size: Option<i32>,
+    pub level_details: Vec<LevelDetail>,
+}
+
+impl Display for Statistics {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        writeln!(f, "File: {}", self.file_path).unwrap();
+        writeln!(f, "FileSize: {:.2}MB", self.file_size).unwrap();
+        writeln!(f, "Schema: {}", self.schema).unwrap();
+        writeln!(f, "Page size: {} bytes", self.page_size.unwrap()).unwrap();
+        writeln!(
+            f,
+            "|{:^9}|{:^9}|{:^9}|{:^9}|{:^9}|{:^9}|",
+            "zoom", "count", "smallest", "largest", "average", "bbox"
+        )
+        .unwrap();
+
+        for l in &self.level_details {
+            let bbox_str = format!(
+                "{:.2}, {:.2}, {:.2}, {:.2}",
+                l.bounding_box[0], l.bounding_box[1], l.bounding_box[2], l.bounding_box[3]
+            );
+
+            writeln!(
+                f,
+                "|{:^9}|{:^9}|{:^9}|{:^9}|{:^9}|{:^9}|",
+                l.zoom,
+                l.count,
+                format!("{:.2}KB", l.smallest),
```

Comment:
this is fine for now, but eventually we will want KB to automatically compute - see https://crates.io/crates/size_format